.azure-pipelines/hpu-tests.yml

```diff
@@ -37,14 +37,17 @@ jobs:
       displayName: 'Install dependencies'
 
     - bash: |
+        export PL_RUN_HPU_TESTS=1
```

Comment:
lets rather use global definition https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#variable-scopes or per step:
https://github.com/PyTorchLightning/pytorch-lightning/blob/b2e98d61661fca80b87e1e2b49cd301d29667ce5/.azure-pipelines/gpu-tests.yml#L117

Reply:
I will be merging them all into one